core/market/decentralized/tests/test_agreement.rs

```diff
@@ -603,7 +657,11 @@ async fn net_err_while_approving() -> Result<()> {
     let result = network
         .get_market(PROV_NAME)
         .provider_engine
-        .approve_agreement(prov_id.clone(), &agreement_id, 0.1)
+        .approve_agreement(
+            prov_id.clone(),
+            &agreement_id.clone().translate(OwnerType::Provider),
```

Comment:
I'd love to see the test for not translating `agreement_id`.  Does it make sense?

Reply:
#510